internal/cmd/cli/create/computeinstance/create_compute_instance_cmd.go

```
@@ -668,9 +694,124 @@ func (c *runnerContext) buildSpec(templateID string,
+		for _, raw := range c.args.networkAttachments {
+			na, err := parseNetworkAttachmentFlag(raw)
+			if err != nil {
+				return err
+			}
+			attachments = append(attachments, na)
+		}
+		spec.NetworkAttachments = attachments
+		return nil
+	}
+	if c.args.subnet != "" {
+		spec.Subnet = proto.String(c.args.subnet)
```

golangci-lint (2.12.2)

[error] SA1019: spec.Subnet is deprecated: use network_attachments. Single subnet for the VM's primary NIC.

(staticcheck)